web/book/src/reference/syntax/strings.md

```diff
@@ -60,3 +60,23 @@ and PRQL doesn't currently account for
 these differences. Please open issues with any difficulties in the current
 implementation.
 ```
+
+## Escape sequences
+
+Unless an `'r'` prefix is present, escape sequences in string literals are
+interpreted according to rules similar to those used by Standard C. The
+recognized escape sequences are:
+
+| Escape Sequence    | Meaning                       |
+| ------------------ | ----------------------------- |
+| `\`&lt;newline&gt; | Backslash and newline ignored |
```

Comment:
I don't completely get this one — should it be `\\n`?

Reply:
Oh, it was meant for string continuation on the next line, but PRQL doesn't have this since strings can span multiple lines anyway without a continuation token, so I removed it.